scripts/refunder.ts

```diff
@@ -0,0 +1,32 @@
+
+import hre from 'hardhat'
+const ethers = hre.ethers;
+
+async function refunder(factoryAddress: string, masterRefunderAddress: string, refunderVersion = 1) {
+
+	// Compile our Contracts, just in case
+	await hre.run('compile');
+
+	if (!factoryAddress) {
+		throw Error('Missing FACTORY ADDRESS');
+	}
+
+	if (!masterRefunderAddress) {
+		throw Error('Missing MASTER REFUNDER ADDRESS');
+	}
+
+	const factory = await ethers.getContractAt("RefunderFactory", factoryAddress);
+
+    let res = await factory.createRefunder(masterRefunderAddress, refunderVersion);
+	let txReceipt = await res.wait();
+	
+	if (txReceipt.events.length > 0) {
+		const newRefunderAddress = txReceipt.events[2].args.refunderAddress;
+  		console.log("Refunder deployed to:", newRefunderAddress);
+	} else {
+		console.log(`Tx hash:`, res.hash);
+	}
```

Comment:
If we change the refunder logic (add/remove event) this will be broken. Lets do `txReceipt.events.Filter` and search for our event and not hardcode `2`